### DISCUSSION TITLE
Uninstall Procedure

### DISCUSSION THREAD

**Discussion by claughinghouse** [lima-vm/lima#463] [Q&A] [answered]
To uninstall Lima the standard `brew remove lima` works to uninstall the cli tools but leaves the hypervisor running and unable to be managed. 

Is there a way to uninstall and cleanup downloaded images, terminate running VMs, and leave the system in a state prior to install?

**Comment by AkihiroSuda** [comment, marked as answer]
```
limactl rm -f $(limactl ls -q)
rm -rf ~/.lima ~/Library/Caches/lima
brew uninstall lima
```
